packages/components/src/components/CollapsibleBox/CollapsibleBox.tsx

```diff
@@ -17,140 +17,86 @@ const animationVariants = {
 `;
 
 const IconLabel = styled.div`
-    margin-right: 6px;
-    margin-left: 28px;
-    color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
-    font-size: ${variables.FONT_SIZE.SMALL};
-    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    margin-right: ${spacingsPx.sm};
+    color: ${({ theme }) => theme.textSubdued};
+    ${typography.hint}
 `;
 
 const Heading = styled.span<Pick<CollapsibleBoxProps, 'variant'>>`
     display: flex;
     align-items: center;
-    color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
-    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    ${typography.body}
+`;
 
-    ${({ variant }) =>
-        variant === 'tiny' &&
-        css`
-            font-size: ${variables.NEUE_FONT_SIZE.NANO};
-        `}
+const SubHeading = styled.span`
+    ${typography.hint}
+    color: ${({ theme }) => theme.textSubdued};
+`;
 
-    ${({ variant }) =>
-        (variant === 'small' || variant === 'large') &&
-        css`
-            font-size: ${variables.NEUE_FONT_SIZE.SMALL};
-        `}
+const easingValues = motionEasing.transition.join(', ');
+const ANIMATION_DURATION = 0.4;
+const StyledIcon = styled(Icon)<{ isCollapsed?: boolean }>`
+    transform: ${({ isCollapsed }) => (isCollapsed ? 'rotate(0deg)' : 'rotate(180deg)')};
+    transition: transform ${ANIMATION_DURATION}s cubic-bezier(${easingValues});
+    transform-origin: center;
 `;
 
 const Content = styled.div<{
     variant: CollapsibleBoxProps['variant'];
-    $noContentPadding?: boolean;
 }>`
     display: flex;
     flex-direction: column;
+    padding: ${({ variant }) =>
+        variant === 'small'
+            ? `${spacingsPx.lg} ${spacingsPx.md}`
+            : `${spacingsPx.xl} ${spacingsPx.md}`};
+    border-top: 1px solid ${({ theme }) => theme.borderOnElevation1};
     overflow: hidden;
-    border-top: 1px solid ${({ theme }) => theme.STROKE_GREY};
-
-    ${({ $noContentPadding, variant }) =>
-        !$noContentPadding &&
-        variant === 'tiny' &&
-        css`
-            padding: 15px 16px;
-        `}
-
-    ${({ $noContentPadding, variant }) =>
-        !$noContentPadding &&
-        variant === 'small' &&
-        css`
-            padding: 20px 16px;
-        `}
-
-    ${({ $noContentPadding, variant }) =>
-        !$noContentPadding &&
-        variant === 'large' &&
-        css`
-            padding: 20px 30px;
-        `}
 `;
 
 const Collapser = styled(motion.div)`
     overflow: hidden;
 `;
 
-interface CollapsibleBoxProps extends React.HtmlHTMLAttributes<HTMLDivElement> {
-    heading: React.ReactNode;
-    variant: 'tiny' | 'small' | 'large';
+export interface CollapsibleBoxProps {
+    heading?: React.ReactNode;
+    subHeading?: React.ReactNode;
+    variant: 'small' | 'large'; // TODO: reevaluate variants
     iconLabel?: React.ReactNode;
-    children?: React.ReactNode;
-    noContentPadding?: boolean;
-    headerJustifyContent?: 'space-between' | 'center';
-    opened?: boolean;
+    isOpened?: boolean;
```

Comment:
```suggestion
    isOpen?: boolean;
```

seems to be the more common variant in our code (nit)